codegen/src/main/twirl/templates/ScalaServer/PowerApiTrait.scala.txt

```diff
@@ -0,0 +1,27 @@
+@*
+ * Copyright (C) 2018 Lightbend Inc. <https://www.lightbend.com>
+ *@
+
+@(service: akka.grpc.gen.scaladsl.Service)
+
+@akka.grpc.gen.Constants.DoNotEditComment
+package @service.packageName
+
+import scala.concurrent.Future
+
+import akka.NotUsed
+import akka.stream.scaladsl.Source
+import akka.grpc.scaladsl.MetadataMap
+import akka.grpc.GrpcServiceException
+
+import io.grpc.Status
+
+trait @{service.name}PowerApi extends @{service.name} {
```

Comment:
In the future we'd want to allow finer-grained configuration to determine which methods should have the power API and which methods have the simple API. In that case perhaps `PowerApi` is not a great name - though I'm not sure about alternatives. This is OK for now.